app/models/fraud/indicator.rb

```diff
@@ -111,6 +117,7 @@ def riskylist
     end
 
     def calculate_points_from_count(count)
+      return 0 if threshold.present? && count < threshold
```

Comment:
This ensures that we only attribute points for a duplicate infraction if the number of infractions is greater than the indicated threshold